cli/cmd/env/rm.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // RmCmd is sub command for env. Removes the variable for a project
 func RmCmd(ch *cmdutil.Helper) *cobra.Command {
```

Comment:
The `rm` command from older CLIs will no longer work. As per the older behaviour it will delete removed variable and send remaining variable.
The only option I see it to create a `v2` API.

Reply:
It's not a commonly used command, so it's okay to just require folks to upgrade. You can reject requests to the RPC from old CLIs using this: https://github.com/rilldata/rill/blob/2396d5c36384a911502db944d3f7a4142738c5d1/admin/server/server.go#L41

Reply:
Sounds good. 
But the same method is also used in `rill env configure` which is somewhat essential for most deploys. This means most users will need to upgrade their CLIs but that should be okay since we handle it gracefully now.